src/tlog-test-json-stream.c

```diff
@@ -232,52 +233,65 @@ test(const char *n, const struct test t)
             if ((buf < op->data.write.buf) ||
                 (buf - op->data.write.buf) !=
                     (ssize_t)(op->data.write.len_in -
-                              op->data.write.len_out))
+                              op->data.write.len_out)) {
                 FAIL_OP("off %zd != %zu",
                         (buf - op->data.write.buf),
                         (op->data.write.len_in - op->data.write.len_out));
-            if (len != op->data.write.len_out)
+            }
+            if (len != op->data.write.len_out) {
                 FAIL_OP("len %zu != %zu", len, op->data.write.len_out);
-            if ((meta.ptr - last_ptr) != op->data.write.meta_off)
+            }
+            if ((meta.ptr - last_ptr) != op->data.write.meta_off) {
                 FAIL_OP("meta_off %zd != %zd",
                         (meta.ptr - last_ptr), op->data.write.meta_off);
+            }
             last_ptr = meta.ptr;
             if (((ssize_t)last_rem - (ssize_t)meta.rem) !=
-                    op->data.write.rem_off)
+                    op->data.write.rem_off) {
                 FAIL_OP("rem_off %zd != %zd",
                         (last_rem - meta.rem), op->data.write.rem_off);
+            }
             last_rem = meta.rem;
-            if (passed)
+            if (passed) {
                 break;
-            else
+            }
+            else {
```

Comment:
Could you please put else and braces on a single line, like this:

    } else {